pyproject.toml

```diff
@@ -38,6 +38,13 @@ dev = [
     "pytest-cov",
 ]
 
+[project.scripts]
+ppafm-generate-elff = "ppafm.cli.GenerateELFF:main"
+ppafm-generate-elff-point-charges = "ppafm.cli.GenerateELFF_point_charges:main"
+ppafm-generate-ljff = "ppafm.cli.GenerateLJFF:main"
+ppafm-relaxed-scan = "ppafm.cli.relaxed_scan:main"
+ppafm-plot-results = "ppafm.cli.plot_results:main"
+
```

Comment:
Can we also add the GUI here from `GUI/ppm-gui` and maybe rename it to `ppafm-gui`? The only problem is that it requires the optional `[opencl]` dependencies.

Reply:
done in 5bb1627